lib/gui/app.js

```diff
@@ -140,17 +140,17 @@ app.run((AnalyticsService, OSWindowProgressService, FlashStateModel) => {
     //
     // We use the presence of `.eta` to determine that the actual
     // writing started.
-    if (!FlashStateModel.isFlashing() || !flashState.eta) {
+    if (!flashState.isFlashing() || !flashState.eta) {
```

Comment:
See https://github.com/resin-io/etcher/pull/1273. Sorry about that, I got confused by the `flashState`/`currentFlashState` naming and assumed that `flashState` was the desired state.